sftp/1.4/modules/ROOT/pages/sftp-write.adoc

```diff
@@ -28,13 +28,13 @@ In the XML editor, the `<sftp:write>` configuration looks like this:
 
 == Use Embedded DataWeave Transformations
 
-If you place a Transform component before the Write operation, the message payload changes, and that impacts the operation that is placed after the Write operation.
-Therefore, if the payload is in a different format and you need to transform it before writing it, you can place the transformation inside the *Content* parameter of the Write operation to generate content to be written without affecting the message in transit.
+If you place a Transform Message component before the Write operation, the message payload changes, and that impacts the operation that is placed after the Write operation.
```

Comment:
Per our style unless the rest of the doc doesn't have this: *Transform Message*
*Write* operation